test/go.mod

```diff
@@ -1,25 +1,27 @@
 module github.com/metal3-io/baremetal-operator/test
 
-go 1.20
+go 1.21
+
+toolchain go1.21.11
```

Comment:
Can we prevent the bot from doing this? I guess we probably don't care much since it's a stable branch..

Reply:
This will be removed from this PR when the manual go bump is merged.